stest/assertions.go

```diff
@@ -43,12 +43,21 @@ func AssertExactMatch(t *testing.T, evs []s.Event, preds []EventP) {
 	}
 }
 
-// verifyPartialMatch is an utility function that matches in order a list of
-// EventP predicates to a list of supervision system events. The input events
-// need to match in order, but the events do not need to be a 1 to 1 match (e.g.
-// the input events slice length may be bigger than the predicates slice
-// length). This function is useful when we want to test that _some_ events are
-// present in the expected order in a noisy system.
+// verifyPartialMatch is a utility function that matches (in order) a list of
+// EventP predicates to a list of supervision system events.
+//
+// The supervision system events need to match in order all the list of given
+// predicates, however, there has not to be a one to one match between the input
```

Comment:
Very minor wording/editorial suggestion: s/_there has not to be_/_there does not need to be_/